### DISCUSSION TITLE
Making changes to schema without NocoDB GUI - tables not appearing ? [Apr 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1678] [Discord Threads]
> 🙋‍♂️ **Harbard**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/sync-schema  
#  
> 🙋‍♂️ **Harbard**  
#### Thanks, but I already went to this tab and is says `Tables metadata is in sync`  
#  
> 💁 **o1lab**  
#### pls project info  
#  
> 💁 **o1lab**  
#### if its on root db - it shud be prefixed with the project id  
#  
> 💁 **o1lab**  
#### you can identify this by creating a 'test' table from nocodb gui  
#  
> 💁 **o1lab**  
#### and see the name under which it saves it  
#  
> 💁 **o1lab**  
#### project info - can be got from right top menu  
#  
> 🙋‍♂️ **Harbard**  
#### Oh, alright I see, thank you very much! <:Prayge:846862909846061067> I guess I'll move the root db in another DB then, I'll keep things more readable !  
#  
> 🙋‍♂️ **Harbard**  
#### Hum, I tried to create a project by connecting it to a `sample_db` DB and it created a `sample_db__noco_1` DB instead. I just followed https://www.youtube.com/watch?v=tb5UyaQns4A  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/960825020173668362)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
